package.json

```
@@ -19,7 +19,7 @@
     "typescript": "^3.8.3"
   },
   "scripts": {
-    "prepare": "npm run build:clean; npm run build",
+    "prepare": "(npm run build:clean || true) && npm run build",
```

What is the purpose for `|| true`? I don't think `npm run build:clean` can ever fail. I'm not the best at some of these commands tho. Especially the difference between `&&`, `;`, etc.

---

This is aim to add windows support for `npm install hardyscc/dynamoose --save`

Please see: 
https://medium.com/@chillypenguin/running-node-npm-scripts-sequentially-on-windows-8737dc24da1f

---

@hardyscc Ahh got it. It makes sense for `&&` but not `||`. What happens if we change it to something like the following? Just since I don't think `npm run build:clean` can ever fail (and if it does, we have much bigger problems).

```suggestion
    "prepare": "npm run build:clean && npm run build",
```

---

Yes sure can change to that,
this is just in case clean fail, the latter still run.